docs/in-the-media/links.md

```diff
@@ -1,5 +1,12 @@
 # Articles and news
 
+**February 27, 2025**
+
+*Manfred Moser* and *Cole Bowden* are joined by *Jan Waś* in Trino Community BroadCast 71: Fake it real good to learn about the new Faker connector and the Datafaker library. You can find the episode here: 
+
+https://trino.io/broadcast/
```

Comment:
Adjust to link to the actual episode so over time it does not become an invalid link -p 

```suggestion
https://trino.io/broadcast/71.html
```